### ISSUE TITLE
Entity generation for partitioned tables

### ISSUE THREAD

**Issue by jared-mackey** [SeaQL/sea-orm#1582]
## Description
Hi,

I'm not certain this is a bug but I don't know if it's correct behavior either. When generating entities for my schema I noticed that the entity generator does a few unexpected things with my partitioned tables:

- It tries to generate an entity for each partition instead of just one for the super table.
  - I was able to ignore these though via the CLI parameters.
- Entity relations are being created for table that FK to the super table but one per partition.

## Steps to Reproduce

1. Create a partitioned table. Let's call this `table_1` and then each partition will be `table_1_p1` through `table_1_p128`.
2. Create a normal table that FKs to the partition table. Let's call this `table_2`.
3. Generate the entities with `sea-orm-cli generate entity -o entity/src --lib --tables "table_1,table_2" --ignore-tables "table_1_p1,table_1_p2..." --max-connections 25`

### Expected Behavior

- I would not get entities created for `table_1_p1`..`table_1_p128`.
- Entity for `table_2` would have a single relationship defined -- the one to `table_1`.

### Actual Behavior

✅ I did not get entities created for `table_1_p1` 
✅ The `table_2` entity correctly had a FK to the main table.
❌ The `table_2` entity was _filled_ with relationships to `table_1_p1`..`table_1_p128`.

Example

```rs
#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
pub enum Relation {
    #[sea_orm(
        belongs_to = "super::table_1_p34::Entity", // <-- actually doesn't exist because I don't generate those entities
        from = "Column::Table1Id",
        to = "super::table_1_p34::Column::Id", // <-- same
        on_update = "NoAction",
        on_delete = "NoAction"
    )]
    Table1P34,
...
}

impl Related<super::table_2_p34::Entity> for Entity {
    fn to() -> RelationDef {
        Relation::Table1P34.def()
    }
}
```

### Reproduces How Often

Every time.

### Workarounds

Delete the wrongly generated code.

## Versions

0.11.2

**Comment by tyt2y3** [comment]
Can you create a new folder under `/issues`, and attach a minimal schema?
Schema discovery did not take in consideration partitioned tables, so we'll need to figure out how to identify and remove the duplicates.

**Comment by cemremengu** [comment]
As a note, other libraries like jooq (java) provide a regex mechanism to exclude unwanted tables like partitions in a schema such as `<excludes>.*_p[0-9_]+</excludes>`

**Comment by billy1624** [comment]
Hey @jared-mackey, thanks for the report! I just drafted a PR to skip parsing partitioned Postgres tables.

**Comment by billy1624** [comment]
The patch will be released in the coming 0.12.x
